pulp_python/app/viewsets.py

```diff
@@ -112,17 +113,17 @@ class PythonRemoteViewSet(platform.RemoteViewSet):
     """
     A ViewSet for PythonRemote.
 
-    Similar to the PythonContentViewSet above, define endpoint_name,
-    queryset and serializer, at a minimum.
+    Similar to the PythonContentViewSet above, define endpoint_name, queryset and serializer,
+    at a minimum.
     """
 
     endpoint_name = 'python'
     queryset = python_models.PythonRemote.objects.all()
     serializer_class = python_serializers.PythonRemoteSerializer
     filter_class = PythonRemoteFilter
 
-    @decorators.detail_route(methods=('post',),
-                             serializer_class=python_serializers.PythonSyncSerializer)
+    @decorators.detail_route(
+        methods=('post',), serializer_class=python_serializers.PythonSyncSerializer)
```

Comment:
Here's an example of where it just looks a lot nicer on the same line though.  I don't feel strongly about making it perfectly consistent vs. looking clean.

Reply:
+1 to this way when it is only 1 line